src/main/java/seedu/address/logic/commands/wallet/WalletExpenseCommand.java

```diff
@@ -49,7 +49,11 @@ public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
 
         model.addExpense(toAdd);
-        return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
+
+        System.out.println(String.format(MESSAGE_SUCCESS, toAdd,
+                model.getTotalExpenditureInMonth(toAdd.getDate()), model.getBudget(toAdd.getDate())));
```

Comment:
[Question]:

Instead of using sysout will using Logger be better? Or is this sysout just or checking purpose?

Reply:
oops this is for checking, will remove!